irods/data_object.py

```diff
@@ -46,10 +46,13 @@ def __repr__(self):
 
 class iRODSDataObject:
 
-    def __init__(self, manager, parent=None, results=None):
+    def __init__(self, manager, /, parent=None, results=None, *, results_sort_key = None):
```

Comment:
Will the `/` break this interface? In other words, if a client is calling this with `manager` as a keyword argument, will this cause it to experience errors? I doubt anybody is doing that, but just curious since this will be going into a minor release.